README.md

```diff
@@ -173,16 +173,18 @@ npm build
 serve -s build -l 80
 ```
 
-This will serve the build folder on port 80 of your machine meaning it can be retrevied via a browser by either going to your machines public IP or hostname
+The above will serve the build folder on port 80.
+
+View Lightspeed in your web browser by visiting http://hostname or http://your.ip.address.here
 
 
 <!-- _For more examples, please refer to the [Documentation](https://example.com)_ -->
 
 ## Streaming From OBS
 
-By default since we are using the FTL protocol you cannot just use a custom server. You will need to edit your `services.json` file. It can be found at `%AppData%\obs-studio\plugin_config\rtmp-services\services.json` on Windows and `/Users/YOURUSERNAME/Library/Application\ Support/obs-studio/plugin_config/rtmp-services/services.json`
```

Comment:
This is backwards. The %AppData% one is Windows and the /Users is Mac

Reply:
Good spot.